distros/ArchLinux/mkconfig.sh

```diff
@@ -7,5 +7,5 @@ sed -i '/^post_install/r../../dist-utils/common/postinstall' "$DEST_FILE"
 sed -i '/^pre_remove/r../../dist-utils/common/preremove' "$DEST_FILE"
 sed -i '2,${/^#!/d}' "$DEST_FILE"
 
-export COPY_CMD='cp *.pkg.tar.xz /app/'
```

Comment:
The extension is set on purpose in order to be predictable an this way we can upload the package. If you change the extension it must be changed also here https://github.com/Stremio/stremio-shell/blob/master/.travis.yml#L64

Reply:
Done. Thank you for the instructions!

Reply:
Thank you for the PR. Obviously we have some issues with CI and the build fails. When we fix it we will merge your code.